<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-0E4BT9V" failures="0" tests="6" name="com.mailchimp.automation.testcases.SignUpPageTest" time="0.003" errors="1" timestamp="10 Aug 2018 10:47:52 GMT" skipped="5">
  <testcase classname="com.mailchimp.automation.testcases.SignUpPageTest" name="signUpPageTitleTest" time="0.000">
    <skipped/>
  </testcase> <!-- signUpPageTitleTest -->
  <testcase classname="com.mailchimp.automation.testcases.SignUpPageTest" name="setUp" time="0.002">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.mailchimp.automation.suite.TestHelper.setUpDriver(TestHelper.java:37)
	at com.mailchimp.automation.suite.TestHelper.getCurrentDriver(TestHelper.java:95)
	at com.mailchimp.automation.pages.PageBase.<init>(PageBase.java:11)
	at com.mailchimp.automation.pages.SignUpPage.<init>(SignUpPage.java:48)
	at com.mailchimp.automation.testcases.SignUpPageTest.setUp(SignUpPageTest.java:22)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase classname="com.mailchimp.automation.testcases.SignUpPageTest" name="logInLinkTest" time="0.000">
    <skipped/>
  </testcase> <!-- logInLinkTest -->
  <testcase classname="com.mailchimp.automation.testcases.SignUpPageTest" name="signUp_With_Invalid_Email_Test" time="0.000">
    <skipped/>
  </testcase> <!-- signUp_With_Invalid_Email_Test -->
  <testcase classname="com.mailchimp.automation.testcases.SignUpPageTest" name="signUp_With_Invalid_Email_And_Existing_UserName_Test" time="0.000">
    <skipped/>
  </testcase> <!-- signUp_With_Invalid_Email_And_Existing_UserName_Test -->
  <testcase classname="com.mailchimp.automation.testcases.SignUpPageTest" name="signUp_With_Valid_Email_And_UserName_Test" time="0.001">
    <skipped/>
  </testcase> <!-- signUp_With_Valid_Email_And_UserName_Test -->
</testsuite> <!-- com.mailchimp.automation.testcases.SignUpPageTest -->
